client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -819,6 +816,45 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
         }
     }
 
+    const isEqual = (first: any, second: any) => {
```

Comment:
All of this logic here is very confusing. What is the purpose of this code?

Reply:
This question includes the titleByAdminLandingFilter and tableTitle code.

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1052/commits/2919164a13b53bd96d406429a5c24094eb3ab12d)